src/components/forms/SignUpForm/index.tsx

```diff
@@ -1,9 +1,12 @@
-import React from 'react';
+import React, { useState } from 'react';
```

Comment:
I think that 'React' can be removed from the imports
```suggestion
import { useState } from 'react';
```

Reply:
When I tried to remove the 'React' import, I get the follow error:

<img width="744" alt="Screen Shot 2021-12-27 at 11 12 42 AM" src="https://user-images.githubusercontent.com/17410581/147489063-8dd9ae6e-8af1-4ea8-a9a1-40befc662d56.png">

Reply:
Ok, we'll leave it as it is!